pkg/agent/openflow/client.go

```diff
@@ -39,6 +39,25 @@ import (
 
 const maxRetryForOFSwitch = 5
 
+const (
+	TrafficControlFlowPriorityHigh   types.TrafficControlFlowPriority = "high"
+	TrafficControlFlowPriorityMedium types.TrafficControlFlowPriority = "medium"
+	TrafficControlFlowPriorityLow    types.TrafficControlFlowPriority = "low"
```

Comment:
To avoid cycled-import in test code, I only moved the definition to another file. It's okay to move these definitions to the where   `TrafficControlFlowPriority` is defined.